src/test/java/org/sagebionetworks/bridge/hibernate/HibernateAccountTest.java

```diff
@@ -4,38 +4,140 @@
+        account.setRoles(ImmutableSet.of(DEVELOPER, RESEARCHER));
+        account.setStatus(ENABLED);
+        account.setClientData(TestUtils.getClientData());
+        account.setVersion(1);
+        account.setTimeZone(DateTimeZone.UTC);
+        account.setDataGroups(USER_DATA_GROUPS);
+        account.setLanguages(LANGUAGES);
+        account.setReauthToken("reauthToken");
+        
+        // Do this just to verify it is not included in the serialization.
+        addConsentHistories(account);
+        
+        JsonNode node = BridgeObjectMapper.get().valueToTree(account);
+        assertEquals(node.get("type").textValue(), "Account");
+        assertEquals(node.size(), 19);
+        assertEquals(node.get("id").textValue(), "id");
+        assertEquals(node.get("orgMembership").textValue(), "orgId");
+        assertEquals(node.get("email").textValue(), "email");
+        assertEquals(node.get("synapseUserId").textValue(), "synapseUserId");
+        assertEquals(node.get("phone").get("nationalFormat").textValue(), PHONE.getNationalFormat());
+        assertEquals(node.get("phone").get("regionCode").textValue(), PHONE.getRegionCode());
+        assertTrue(node.get("emailVerified").booleanValue());
+        assertTrue(node.get("phoneVerified").booleanValue());
+        assertEquals(node.get("attributes").get("a").textValue(), "b");
+        assertEquals(node.get("attributes").get("c").textValue(), "d");
+        assertEquals(node.get("createdOn").textValue(), CREATED_ON.toString());
+        assertEquals(node.get("modifiedOn").textValue(), MODIFIED_ON.toString());
+        assertEquals(node.get("firstName").textValue(), "firstName");
+        assertEquals(node.get("lastName").textValue(), "lastName");
+        assertEquals(toSet(node, "roles"), ImmutableSet.of("developer", "researcher"));
+        assertEquals(node.get("status").textValue(), "enabled");
+        assertNotNull(node.get("clientData"));
+        assertEquals(node.get("version").intValue(), 1);
+        assertEquals(toSet(node, "dataGroups"), ImmutableSet.of("group1", "group2"));
+        assertEquals(toSet(node, "languages"), ImmutableSet.of("en", "fr"));
+        
+        // these should be null
+        assertNull(node.get("appId"));
+        assertNull(node.get("consents"));
+        assertNull(node.get("healthCode"));
+        assertNull(node.get("passwordAlgorithm"));
+        assertNull(node.get("passwordHash"));
+        assertNull(node.get("passwordModifiedOn"));
+        assertNull(node.get("reauthToken"));
+        assertNull(node.get("timeZone"));
+        assertNull(node.get("sharingScope"));
+        assertNull(node.get("notifyByEmail"));
+        assertNull(node.get("migrationVersion"));
+        assertNull(node.get("enrollments"));
+        assertNull(node.get("reauthToken"));
+        assertNull(node.get("activeConsentSignature"));
+        assertNull(node.get("consentSignatureHistory"));
+        assertNull(node.get("allConsentSignatureHistories"));
```

Comment:
We should add these attributes to the Accounts object so we can distinguish between the field not being present and the field being ignored by Jackson.

Also, we're missing getActiveEnrollments.